client/src/components/BottomNavigationBar.js

```diff
@@ -0,0 +1,62 @@
+import React, { useState } from "react";
+import { BottomNavigation, BottomNavigationAction } from "@material-ui/core";
+import { makeStyles } from "@material-ui/core/styles";
+import { ChatBubbleOutline } from "@material-ui/icons";
+import { ReactComponent as ChefLogo } from "images/chefs_hat_icon.svg";
+import { ReactComponent as DishIcon } from "images/dish_icon.svg";
+import UserAvatar from "common/UserAvatar";
+
+const BottomNavigationBar = ({ loggedInUser }) => {
+  const classes = useStyles();
+  const [value, setValue] = useState(0);
+
+  return (
+    <BottomNavigation
+      value={value}
+      onChange={(event, newValue) => {
+        setValue(newValue);
+      }}
+      showLabels
+      className={classes.stickToBottom}
+    >
+      <BottomNavigationAction
+        label="Chefs"
+        value="chefs"
+        icon={<ChefLogo className={classes.icon} />}
+      />
+      <BottomNavigationAction
+        label="Dishes"
+        value="dishes"
+        icon={<DishIcon className={classes.icon} />}
+      />
+      <BottomNavigationAction
+        label="Messages"
+        value="messages"
+        icon={<ChatBubbleOutline className={classes.icon} />}
+      />
+      <BottomNavigationAction
+        label="Profile"
+        value="profile"
+        icon={<UserAvatar className={classes.avatarIcon} user={loggedInUser} />}
+      />
```

Comment:
I feel like we should use a .map here to go through an array of objects, so this is more easy to edit in the future

Reply:
An array of what exactly ?

Reply:
Something like:
```
[{"label": "Dishes", "value": "dishes", "icon": DishIcon}, ...
```
Will reduce a bit of the code and is more easy to edit